### ISSUE TITLE
Keystroke for BlockToolbar

### ISSUE THREAD

**Issue by oskarwrobel** [ckeditor/ckeditor5-ui#419]
It is possible to navigate through block toolbar items using <kbd>arrow left</kbd> and <kbd>arrow right</kbd> keys. It is possible to close the block toolbar using <kbd>Esc</kbd> or moving focus back to the editable using <kbd>Tab</kbd>. The only missing feature is keystrok for opening and focusing the block toolbar.

![jun-27-2018 20-39-18](https://user-images.githubusercontent.com/1057056/41992948-3784a946-7a4a-11e8-80f0-7954079cc9ac.gif)

**Comment by Reinmar** [comment]
What about <kbd>Tab</kbd>? We use it now to access the link balloon. Maybe we could also use it to access the balloon toolbars? Or would that be weird?

**Comment by oleq** [comment]
> What about Tab? We use it now to access the link balloon. Maybe we could also use it to access the balloon toolbars? Or would that be weird?

<kbd>Tab</kbd> will collide with table (next cell) and list (indent) keystrokes. Unless it's a UI attached to the selection (like link), it does not make much sense IMO.

But maybe explain your idea because it's super vague. What would it work like?

**Comment by Reinmar** [comment]
> Tab will collide with table (next cell) and list (indent) keystrokes. Unless it's a UI attached to the selection (like link), it does not make much sense IMO.

Good point. I completely forgot about this. So I take that back – <kbd>Tab</kbd> makes no sense here.
